datatypes/src/operations/reproject.rs

```diff
@@ -229,7 +229,8 @@ where
 
         let proj_outline_coordinates = projector.project_coordinates(&outline_coordinates)?;
 
-        Ok(MultiPoint::new_unchecked(proj_outline_coordinates).spatial_bounds())
+        let bbox = MultiPoint::new_unchecked(proj_outline_coordinates).spatial_bounds();
+        A::from_min_max(bbox.lower_left(), bbox.upper_right())
```

Comment:
i see why you use from_min_max everywhere. However, cant we have a from_ll_ur? i think it would be more expressive.

Reply:
to make it more expressive I would have to call it `from_lower_left_upper_right` I guess but thats somewhat long.

Reply:
yes but the is min_max the same as ll_ur? or will it reorder so that it fits?

Reply:
I enhanced the function doc to make it more clear. shall I still change the name? e1409767f7d3ce765dc1529071554721b50af98b

Reply:
The function is still not 100% clear to me.

Is from_min_max the minimum enclosing rectangle for coordinates a and b or is the max taken from b and min from a?

Reply:
it takes two coordinates, the minimum aka lower left and the maximum aka upper right. it doesnÄt find the min/max of all the coordinates.